tests/system/test_client.py

```diff
@@ -859,6 +859,67 @@ def test_load_table_from_json_basic_use(self):
         self.assertEqual(tuple(table.schema), table_schema)
         self.assertEqual(table.num_rows, 2)
 
+    def test_load_table_from_json_bug_check_with_schema(self):
+        json_rows = [
+            {
+                "age": "18",
+            },
+            {
+                "age": "79",
+            },
+        ]
+
+        dataset_id = _make_dataset_id("bq_system_test")
+        self.temp_dataset(dataset_id)
+        table_id = "{}.{}.load_table_from_json_bug_check_with_schema".format(
+            Config.CLIENT.project, dataset_id
+        )
+
+        # create an empty table with schema
+        table = helpers.retry_403(Config.CLIENT.create_table)(Table(table_id))
```

Comment:
The test name and comment say this table should be created with a schema, but I don't see the schema here.